### PR TITLE
Report generator as submodule debian

### LINKED ISSUE #28: Fetch the report generator code from the repository
The official repository of the report generator is here : https://github.com/savoirfairelinux/asciidoc-report
For now, a static version of this code is kept in the in the report-generator repository. It is a better practice to fetch the code from source every time instead of having a copied version.

### LINKED ISSUE #51: Fix latency tests image import in the report
The latency test part of the report waits for a file named `pub_standalone_interval_between.png`.
This file is not created during the latency test which generates an error in the pdf.

However, two files are created : 
- `pub_results_hypervisor_ci-tool`
- `pub_results_vm_ci-tool`
But are never used in the report.

### PR DESCRIPTION
**Pull request by eroussy**

### CODE REVIEW

File: launch.sh [hunk 1]
```diff
@@ -228,8 +228,13 @@ launch_latency_tests() {
 # Generate the test report and upload it
 generate_report() {
 
-  # Generate pdf
+  # Replace test-report-pdf themes with SEAPATH one
   cd "${WORK_DIR}/ci/test-report-pdf"
+  rm -rf themes
+  mv ../seapath-themes themes
+  mv themes/theme.yml themes/sfl-theme.yml
```
Comment (Paullgk) [inline launch.sh]: Same comment as above

File: launch-yocto.sh [hunk 2]
```diff
@@ -109,8 +109,13 @@ launch_system_tests() {
 # Generate the test report and upload it
 generate_report() {
 
-  # Generate test report
+  # Replace test-report-pdf themes with SEAPATH one
   cd "${WORK_DIR}/ci/test-report-pdf"
+  rm -rf themes
+  mv ../seapath-themes themes
+  mv themes/theme.yml themes/sfl-theme.yml
```
Comment (Paullgk) [inline launch-yocto.sh]: Is there a reason to keep here the sfl theme?

**Review by Paullgk — CHANGES_REQUESTED** [review]

**Review by Paullgk — APPROVED** [review]

**Review by Paullgk — CHANGES_REQUESTED** [review]
In your commit message:

> This prepares the removal of the report-generator directory.
Every file regarding latency results are move out.

Should be:
> This prepare the removal of the report-generator directory.
Every files regarding latency results are move out.

File: launch-yocto.sh (hunk 2 above)
Comment (eroussy) [reply launch-yocto.sh]: The sfl theme is provided by test-report-pdf.
Here, i replace it by the seapath theme, but it is not clear. lets change these lines.

**Review by Paullgk — APPROVED** [review]

**Pull request merged**